lib/abstract_format.ml

```diff
@@ -65,6 +65,7 @@ and expr_t =
   | ExprFun of {line: line_t; name: string option; clauses: clause_t list}
   | ExprLocalCall of {line: line_t; function_expr: expr_t; args: expr_t list}
   | ExprRemoteCall of {line: line_t;  line_remote: line_t; module_expr: expr_t; function_expr: expr_t; args: expr_t list}
+  | ExprIf of {line: line_t; if_clauses: clause_t list}
```

Comment:
Could you change `if_clauses` to `clauses` according to `ExprCase` and `ExprFun`?

Reply:
Fixed it.